app/controllers/application_controller.rb

```diff
@@ -1,4 +1,13 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
+	before_action :configure_permitted_parameters, if: :devise_controller?
 
-end
+    protected
+
+        def configure_permitted_parameters
+            devise_parameter_sanitizer.permit(:sign_up) { |u| u.permit(:firstname, :lastname, :email, :password)}
+
+            devise_parameter_sanitizer.permit(:account_update) { |u| u.permit(:firstname, :lastname, :email, :password, :current_password)}
+		end
+
+end
```

Comment:
needs to configure `sublime` to have extra line.

Reply:
may be you can paste you sublime configuration